docs/glean/metrics.md

```diff
@@ -0,0 +1,24 @@
+<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
+
+# Metrics
+This document enumerates the metrics collected by this project.
+This project may depend on other projects which also collect metrics.
+This means you might have to go searching through the dependency tree to get a full picture of everything collected by this project.
+
+# Pings
+
+ - [deletion_request](#deletion_request)
+
+
+## deletion_request
+This is a built-in ping that is assembled out of the box by the Glean SDK.
+See the Glean SDK documentation for the [`deletion_request` ping](https://mozilla.github.io/glean/book/user/pings/deletion_request.html).
+The following metrics are added to the ping:
+
+| Name | Type | Description | Data reviews | Extras | Expiration |
+| --- | --- | --- | --- | --- | --- |
+| legacy.ids.client_id |[uuid](https://mozilla.github.io/glean/book/user/metrics/uuid.html) |The client id from legacy telemetry.  |[1](https://bugzilla.mozilla.org/show_bug.cgi?id=1635427)||never |
```

Comment:
Make sure that this gets regenerated once you update the `bugs` and `data_review` fields.